lib/views/walks/walks_view.dart

```diff
@@ -246,6 +251,30 @@ class _WalksViewState extends State<WalksView> with WidgetsBindingObserver {
     ScaffoldMessenger.of(context).showSnackBar(snackBar);
   }
 
+  void _news() async {
```

Comment:
If I understand properly, this will keep fetching data from the internet until we have news to display? And then we never fetch again until the app is relaunched.

Wouldn't that be better to put some last_update timestamp somewhere to prevent calling too frequently the endpoint, like we do for walks?

Reply:
True, I could add 'set newsLaunched = true' after the news fetch to prevent further calls to the API. Everytime the widget is rebuild, it would call it again. (Widget is rebuild upon navigating to the widget from the bottomnavigationbar or upon app launch)

I'll follow your recommendation and add a timestamp which will call for news once every day.